FMI2QGIS/core/processing/raster_loader.py

```diff
@@ -25,7 +25,10 @@
 from qgis.core import QgsProject, QgsRasterLayer
 
 from ...qgis_plugin_tools.tools.custom_logging import bar_msg
-from ...qgis_plugin_tools.tools.exceptions import QgsPluginNotImplementedException
+from ...qgis_plugin_tools.tools.exceptions import (
+    QgsPluginException,
```

Comment:
This import should be removed to satisfy the pre-commit.

Reply:
Got it, thanks!